.github/workflows/fill_station_action.yml

```diff
@@ -1,135 +1,135 @@
-name: Create and publish the Fill Station Docker image
+# name: Create and publish the Fill Station Docker image
```

Comment:
I don't love commenting this out. Should be a Github level change to make it not required. 

In Settings->Rules->Rulesets->PR Reviews and Checks->Require status checks to pass Additional Settings->Delete build-and-test (arm64)